problemsets-mat223/modules/module12.tex

```diff
@@ -85,8 +85,18 @@
 \begin{example}
 	Let $\mathcal P:\R^2\to\R^2$ be projection onto the $x$-axis and let $\mathcal R:\R^2\to\R^2$ be rotation counter-clockwise
 	by $15^\circ$. Classify each of $\mathcal P$ and $\mathcal R$ as invertible or not.
-
-	XXX Finish
+	
+	Since vectors on y-axis projected onto the x-axis are $\vec 0$, we know
```

Comment:
It's more convincing to give specific examples. For instance `\mathcal P\yhat=\mathcal P(2\yhat)=\vec 0$, therefore $\mathcal P$ is not one-to-one